### PR TITLE
Remove periods from titles

### PR DESCRIPTION
**Pull request by jasonblais**
Removing unnecessary periods from title fields.

Also removed it from the "/jira connect" and "/jira disconnect" responses. Since they are embedded links, it looked odd to have the period there.

### CODE REVIEW

File: server/templates/ac/user_connected.html [hunk 1]
```diff
@@ -69,7 +69,7 @@ <h3>
 				<svg xmlns="http://www.w3.org/2000/svg" width="20" height="15" class="icon--check" viewBox="0 0 18 14">
 					<path fill="#0052CC" d="M100.649576,76.1740942 C100.649576,76.4531124 100.537969,76.7321306 100.337075,76.9330237 L90.7388497,86.5312494 C90.5379566,86.7321425 90.2589384,86.8437498 89.9799202,86.8437498 C89.700902,86.8437498 89.4218838,86.7321425 89.2209908,86.5312494 L83.6629484,80.9732071 C83.4620553,80.772314 83.350448,80.4932958 83.350448,80.2142776 C83.350448,79.9352594 83.4620553,79.6562412 83.6629484,79.4553481 L85.1808074,77.9374892 C85.3817005,77.7365961 85.6607186,77.6249888 85.9397368,77.6249888 C86.218755,77.6249888 86.4977732,77.7365961 86.6986663,77.9374892 L89.9799202,81.2299038 L97.3013575,73.8973058 C97.5022506,73.6964127 97.7812688,73.5848054 98.060287,73.5848054 C98.3393052,73.5848054 98.6183234,73.6964127 98.8192165,73.8973058 L100.337075,75.4151648 C100.537969,75.6160579 100.649576,75.895076 100.649576,76.1740942 Z" transform="translate(-83 -73)"/>
 				</svg>
-				Mattermost user is now connected to Jira.
+				Mattermost user is now connected to Jira
```
Comment (jasonblais) [inline server/templates/ac/user_connected.html]: @asaadmahmood Thoughts on updating this to

```
Your Jira account is now connected with Mattermost
```

File: server/templates/ac/user_confirm.html [hunk 2]
```diff
@@ -63,7 +63,7 @@
 	<body>
 		<div class="flex-parent">
 				<h3>
-					Mattermost user is trying to connect to Jira.
+					Mattermost user is trying to connect to Jira
```
Comment (jasonblais) [inline server/templates/ac/user_confirm.html]: Same here, to update to

```
Allow Mattermost to connect to your Jira account
```

File: server/templates/ac/user_connected.html (hunk 1 above)
Comment (asaadmahmood) [reply server/templates/ac/user_connected.html]: @jasonblais Yup, that looks good. Though I am not inline removing the periods.

File: server/templates/ac/user_confirm.html (hunk 2 above)
Comment (asaadmahmood) [reply server/templates/ac/user_confirm.html]: Makes sense.

File: server/templates/ac/user_connected.html (hunk 1 above)
Comment (jasonblais) [reply server/templates/ac/user_connected.html]: @asaadmahmood Ah, interesting. Do we then typically use a period in title page?

I was going based on your original design where a period wasn't used

image

**Comment by jasonblais** [comment]
Updated, awaiting to hear back on the one conversation above @asaadmahmood

**Pull request merged**